packages/types/v1/surveys.ts

```diff
@@ -313,6 +313,7 @@ export const ZSurveyInput = z.object({
 
 export type TSurvey = z.infer<typeof ZSurvey>;
 export type TSurveyInput = z.infer<typeof ZSurveyInput>;
+export type TSurveyInputWithoutIds = Omit<Omit<TSurveyInput, "id">, "environmentId">;
```

Comment:
haha, as regards the first comment I realized `TSurveyInput` was actually never used in the code which is why I omited the environmentId and id since it is create data, what I should have just done is remove the environmentId from `TSurveyInput` and use it here. Tried it now and it works.

For the second, I have noticed `TServiceNameInput` has been a consistent pattern should I stick with this style?

Reply:
sounds good :-) 💪

Reply:
Done